integration/docker/README.md

```diff
@@ -42,6 +42,31 @@ $ docker build -t alluxio/alluxio-dev -f Dockerfile-dev \
 --build-arg JAVA_VERSION=11 .
 ```
 
+To use a customized user/group to launch Alluxio inside containers, build the Dockerfile with
+the user/group with `--build-arg ALLUXIO_USER=`, etc. For example, if you want to use user `alluxio2` with uid `1001` and group
+`alluxio2` with gid `1001`, you need to run the following two commands:
+
+```console
+$ docker build -t alluxio/alluxio:customizedUser \
+--build-arg ALLUXIO_USERNAME=alluxio2 --build-arg ALLUXIO_UID=1001 \
+--build-arg ALLUXIO_GROUP=alluxio2 --build-arg ALLUXIO_GID=1001 .
```

Comment:
Name and ID are both needed?

Reply:
what if i only specify one?

Reply:
This is an example as explained. Not specified ones will use the default values